config/CascadeReweight.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<alg_conf>
+
+<!--
+Configuration sets for the CascadeReweight EventRecordVisitorI
```

Comment:
here add a line saying that clearly this configuration file is done for tuning purposes, and the present configurations are here only as an example